llvm/test/Transforms/LoopVectorize/AArch64/sve-epilog-vect-vscale-tune.ll

```diff
@@ -10,7 +10,7 @@ target triple = "aarch64-unknown-linux-gnu"
 define void @foo(ptr noalias nocapture readonly %p, ptr noalias nocapture %q, i64 %len) #0 {
 ; CHECK-EPILOG:      vec.epilog.ph:
 ; CHECK-EPILOG:      vec.epilog.vector.body:
-; CHECK-EPILOG:        load <vscale x 4 x i16>
+; CHECK-EPILOG:        load <8 x i16>
```

Comment:
I think it's worth adding a new RUN line for neoverse-v1 with the flag `-sve-prefer-fixed-over-scalable-if-equal=false` so that we also test the old behaviour, i.e.

```
  CHECK-EPILOG-PREFER-SCALABLE:        load <vscale x 4 x i16>
```

Reply:
Is the new line relevant for all run lines here? It may be better to have a new test file with a function where the cost of fixed & scalable is the same, which has a comment with a brief explanation and run lines with both the option on and off.

With the changes where its difficult to see if this is working as expected.